website/content/docs/k8s/installation/vault/data-integration/webhook-certs.mdx

```diff
@@ -33,177 +33,159 @@ Complete the following prerequisites prior to implementing the integration descr
-
-
-Finally, Kubernetes auth roles need to be created for controller and connect inject webhooks.
-
-Role for Consul controller webhooks:
-```shell-session
-$ vault write auth/kubernetes/role/controller-role \
-    bound_service_account_names=<Consul controller service account> \
-    bound_service_account_namespaces=<Consul installation namespace> \
-    policies=controller-ca-policy \
-    ttl=1h
-```
-
-To find out the service account name of the Consul controller,
-you can run:
-
-```shell-session
- $ helm template --release-name ${RELEASE_NAME} --show-only templates/controller-serviceaccount.yaml hashicorp/consul
-```
-
-Role for Consul connect inject webhooks:
-
-```shell-session
-$ vault write auth/kubernetes/role/connect-inject-role \
-    bound_service_account_names=<Consul connect inject service account> \
-    bound_service_account_namespaces=<Consul installation namespace> \
-    policies=connect-inject-ca-policy \
-    ttl=1h
-```
-
-To find out the service account name of the Consul connect inject, use the command below.
-```shell-session
- $ helm template --release-name ${RELEASE_NAME} --show-only templates/connect-inject-serviceaccount.yaml hashicorp/consul
-```
-
-### Update the Consul on Kubernetes helm chart
+    -> **Note:** The PKI secret paths referenced by the Vault Policies below will be your `global.secretsBackend.vault.controller.tlsCert.secretName` and `global.secretsBackend.vault.connectInject.tlsCert.secretName` Helm values respectively.
+
+    ```shell-session
+    $ vault policy write controller-tls-policy - <<EOF
+    path controller/issue/controller-role {
+      capabilities = ["create", "update"]
+    }
+    EOF
+    ```
+
+    ```shell-session
+    $ vault policy write connect-inject-policy - <<EOF
+    path connect-inject/issue/connect-inject-role {
+      capabilities = ["create", "update"]
+    }
+    EOF
+    ```
+
+1. Next, create a policy that allows `["read"]` access to the [CA URL](https://www.vaultproject.io/api/secret/pki#read-certificate). The policy is required so that Consul components can communicate with the Consul servers in order to fetch their auto-encryption certificates. Issue the following commands to create the policy:
+
+    -> **Note:** The PKI secret paths referenced by the Vault Policies below will be your `global.secretsBackend.vault.controller.caCert.secretName` and `global.secretsBackend.vault.connectInject.caCert.secretName` Helm values respectively.
+
+    ```shell-session
+    $ vault policy write controller-ca-policy - <<EOF
+    path controller/cert/ca {
+      capabilities = ["read"]
+    }
+    EOF
+    ```
+
+    ```shell-session
+    $ vault policy write connect-inject-ca-policy - <<EOF
+    path connect-inject/cert/ca {
+      capabilities = ["read"]
+    }
+    EOF
+    ```
+1. Issue the following command to create a Vault role for the controller PKI engine and set the default parameters for issuing certificates:
```

Comment:
```suggestion
1. Issue the following command to create a Vault role for the controller PKI engine and set the default parameters for issuing certificates:
```
The way that these steps are described in this page make sense as steps. I added suggestions in server-tls.mdx to use subheadings, but these are similar processes with different wording. Either approach is fine as long as the wording is correct. I suggest either rewording the steps in server-tls.mdx to match the language used in this page, or implementing the subheadings from server-tls.mdx into this page.